content/pt/blog/news/how-to-build-one-effective-middleware.md

```diff
@@ -1,41 +1,41 @@
 ---
-title: "How to build one effective middleware?"
-linkTitle: "How to build one effective middleware?"
+title: "Como construir um intermediário efetivo?"
```

Comment:
You should keep some words in English, like “middleware” instead of “intermediário”, this will help people not to confuse the concepts, and easily find out what they are looking for based on what they already knows.

Reply:
@overlineink The word "intermediário" stands well for "middleware" in English because means "someone or something in the middle" or "something between two edges". I use the same word for the English term in all translation contributions I already done. See my profile.